src/qutip_tensorflow/core/data/norm.py

```diff
@@ -0,0 +1,69 @@
+import qutip
+from .tftensor import TfTensor
+import warnings
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import tensorflow as tf
```

Comment:
A minor comment, but I feel like you may be able to handle this most neatly by having `qutip_tensorflow/__init__.py` do these 3 lines at the very top, and then every internal file just does `import tensorflow`.  Since `tensorflow` will already be imported, I _think_ that the deprecated code inside the module won't be executed, and having it at the top of the main `__init__.py` means it's guaranteed to execute that first.

Reply:
Will try this although I thinks it is better if I do it in another PR together with the rest of files.

Reply:
absolutely, yeah.